### ISSUE TITLE
Use WebView2

### ISSUE THREAD

**Issue by jaigak** [hawkerm/monaco-editor-uwp#64]
WebView 2 will be available to WinUI 2 soon so consider using that instead of EdgeHTML for better performance.

**Comment by hawkerm** [comment]
That's tied to #56 anyway as part of supporting WinUI 3.

**Comment by jaigak** [comment]
> That's tied to #56 anyway as part of supporting WinUI 3.

Right however WinUI 2.8 itself is getting WebView2 so no need of WinUI 3.

**Comment by hez2010** [comment]
Blocking issue: `CoreWebView2.AddHostObjectToScript()` results in AV in UWP.

**Comment by hawkerm** [comment]
@hez2010 is there an issue filed on the WinUI repo for that?

I also hit https://github.com/MicrosoftEdge/WebView2Feedback/issues/75 in general as well which could be challenging to work around, as I'd like new approach to be more direct with some of the Monaco provider objects.

**Comment by hez2010** [comment]
> @hez2010 is there an issue filed on the WinUI repo for that?

No, but I reused the issue https://github.com/microsoft/microsoft-ui-xaml/issues/4377.
